port_ocean/clients/port/mixins/integrations.py

```diff
@@ -197,6 +208,38 @@ async def patch_integration(
         handle_port_status_code(response)
         return response.json()["integration"]
 
+    async def post_integration_metrics(self, metrics: list[dict[str, Any]]) -> None:
+        logger.debug("posting metrics")
+        metrics_attributes = await self.get_metrics_attributes()
+        headers = await self.auth.headers()
+        response = await self.client.post(
+            metrics_attributes["ingestUrl"],
+            headers=headers,
+            json={
+                "kindsMetrics": metrics,
+            },
+        )
+        handle_port_status_code(response, should_log=False)
+        logger.debug("Metrics successfully posted")
+
+    async def put_integration_metrics(self, kind_metrics: dict[str, Any]) -> None:
+        logger.debug("putting metrics")
```

Comment:
lets thrive for more indicative log messages, I am not sure `putting` is the right wording :)